updated with fa 19 final projects (#188)
illinois-cs498vr/illinois-cs498vr.github.io: _projects/FA19-Amusement_Park.md

---
layout: project
title: Amusement Park

> desc: This is a FPS game that simulates amusement park. The player sits in a cart which will move automatically inside the playground. There are two scenes in this game with opposite themes. The first scene is fairy tale scene in which the player will have increasing goal for each level. The second scene is the horror scene. If the player shoot 5 targets then he/she will get level up and targets start to move. If the player is caught by any targets, then GAME OVER...

Reviewer: ```suggestion
desc: This is a FPS game that simulates an amusement park. The player sits in a cart which will move automatically inside the playground. There are two thematically opposite scenes. The first is a fairy tale scene, and the second is a horror scene. If the player shoot 5 targets then he/she will get level up and targets start to move. If the player is caught by any targets, then GAME OVER...
```
